### PR TITLE
Minor things

### PR DESCRIPTION
**Pull request by Goooler**
Closes #135.

### CODE REVIEW

File: diffuse/src/main/rules.txt [hunk 1]
```diff
@@ -18,12 +18,5 @@
...
--keep class com.android.apksig.ApkVerifier$Result {
-    private java.util.List getV4SchemeSigners();
-}
-
-##############
-### APKSIG ###
-##############
--dontwarn org.codehaus.mojo.animal_sniffer.IgnoreJRERequirement
+# Suppress warnings in R8 internal.
+-dontwarn com.android.tools.r8.internal.**
```
Comment (Goooler) [inline diffuse/src/main/rules.txt:22]:
```
Warning in /Users/snebula/IdeaProjects/diffuse/diffuse/build/libs/diffuse-0.2.0-SNAPSHOT-fat.jar:META-INF/services/com.android.tools.r8.internal.WE:
Unexpected reference to missing service class: META-INF/services/com.android.tools.r8.internal.WE.
Warning in /Users/snebula/IdeaProjects/diffuse/diffuse/build/libs/diffuse-0.2.0-SNAPSHOT-fat.jar:META-INF/services/com.android.tools.r8.internal.WE:
Unexpected reference to missing service implementation class in META-INF/services/com.android.tools.r8.internal.WE: com.android.tools.r8.internal.cx.
```

File: renovate.json [hunk 2]
```diff
@@ -2,5 +2,14 @@
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
     "config:base"
+  ],
+  "packageRules": [
+    {
+      "groupName": "AGP",
+      "groupSlug": "agp",
```
Comment (JakeWharton) [inline renovate.json]:
```suggestion
```

This is automatically derived from the name

File: build.gradle [hunk 3]
```diff
@@ -1,15 +1,24 @@
...
 
-  apply plugin: libs.plugins.kotlinter.get().pluginId
+  apply plugin: libs.plugins.spotless.get().pluginId
+  spotless {
+    kotlin {
+      ktlint().editorConfigOverride([
+        'ktlint_standard_filename': 'disabled',
+      ])
+      target('**/*.kt')
+      targetExclude("**/build/**")
```
Comment (JakeWharton) [inline build.gradle:19]: Not sure this is strictly required. Isn't the build directory ignored by default?

**Review by JakeWharton — APPROVED** [review]

File: build.gradle (hunk 3 above)
Comment (Goooler) [reply build.gradle:19]: Oh yes, just copy-paste this from another project🤣.

**Pull request merged**